tests/EventListener/MergeHttpHeadersListenerTest.php

```diff
@@ -227,7 +228,10 @@ public function testHeadersAreInheritedFromSubrequests()
         $this->assertSame('application/json', $response->headers->get('Content-Type'));
     }
 
-    public function testMultiHeadersAreInheritedFromSubrequests()
+    /**
+     * Tests that mulit headers are inherited from a subrequest.
```

Comment:
Typo: `mulit` instead of `multi`.

Reply:
Fixed in 5d4b63390db09308b27c9447d27a687aa35cd0f3